config/database.yml

```diff
@@ -1,15 +1,24 @@
 default: &default
   adapter: postgresql
   encoding: utf8
-  host: localhost
   pool: 10
   timeout: 5000
+  # username: <%= ENV['DB_USERNAME'] %>
+  # password: <%= ENV['DB_PASSWORD'] %>
+  # database: <%= ENV['DB_NAME'] %>
+  # host: <%= ENV['DB_HOST'] %>
+
```

Comment:
I've added it during yesterday's experiments with using ERB to parse this file. Left them over just in case someone needs to use any specific values anywhere in the code.